common/src/main/java/com/microsoft/identity/common/internal/cache/ADALOAuth2TokenCache.java

```diff
@@ -137,7 +140,6 @@ public void saveTokens(
             Logger.infoPII(TAG + ":" + methodName, "issuerCacheIdentifier: [" + issuerCacheIdentifier + "]");
             Logger.infoPII(TAG + ":" + methodName, "scope: [" + scope + "]");
             Logger.infoPII(TAG + ":" + methodName, "clientId: [" + clientId + "]");
-            Logger.infoPII(TAG + ":" + methodName, "cacheItem: [" + cacheItem + "]");
             Logger.infoPII(TAG + ":" + methodName, "cacheIdentifier: [" + cacheIdentifier + "]");
```

Comment:
One small question, what is `cacheIdentifier`?

Reply:
In the `ADALOAuth2TokenCache` there's a `cacheIdentifier` (which is a `userId`) and an `issuerCacheIdentifier` (which corresponds to an authority) -- in the case of `issuerCacheIdentifiers`, there will `n` entries where `n == count of authority aliases`.